doc/examples/planning_scene_ros_api/src/planning_scene_ros_api_tutorial.cpp

```diff
@@ -35,18 +35,18 @@
 /* Author: Sachin Chitta, Michael Lautman */
 
 #include <rclcpp/rclcpp.hpp>
-#include <geometry_msgs/msg/pose.hpp>
+//#include <geometry_msgs/msg/pose.hpp>
```

Comment:
Why are these commented out? I don't think this is related to the ament_target_dependencies deprecation, is this just unrelated cleanup, or am I missing something?